daemon/image_exporter.go

```diff
@@ -9,17 +9,29 @@ import (
 // ExportImage exports a list of images to the given output stream. The
 // exported images are archived into a tar when written to the output
 // stream. All images with the given tag and all versions containing
-// the same tag are exported. names is the set of tags to export, and
-// outStream is the writer which the images are written to.
-func (daemon *Daemon) ExportImage(names []string, outStream io.Writer) error {
-	imageExporter := tarexport.NewTarExporter(daemon.imageStore, daemon.layerStores, daemon.referenceStore, daemon)
+// the same tag are exported. names is the set of tags to export.
+// outStream is the writer which the images are written to. refs is a map used
+// when exporting to the OCI format.
+// format is the format of the resulting tar ball.
+func (daemon *Daemon) ExportImage(names []string, format string, refs map[string]string, outStream io.Writer) error {
+	opts := &tarexport.Options{
```

Comment:
These options are being used as a dependency of `tarexport`, but they are specific to the operation, not re-usable dependencies.

They are also not used by all operations (save uses `Format`, load uses `Name`).
I think they should be split into `SaveOptions` and `LoadOptions`, and should be args to save/load (not the constructor).